workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/BackPressHandler.kt

```diff
@@ -41,41 +44,50 @@ private val View.handlerWrapperOrNull
  * Uses the [androidx.activity.OnBackPressedDispatcher] to associate a [BackPressHandler]
  * with a [View].
  *
- * Registers [handler] whenever [view] is attached to a window, and removes it
- * whenever [view] is detached.
+ * - Registers [handler] on [start]
+ * - Enables and disables it as [view] is attached to or detached from its window
```

Comment:
Because we have no idea if a custom container will remove the view from its parent for a while. w/o this such a view would still consume back press events. Will comment.